iconrpcserver/utils/__init__.py

```diff
@@ -57,3 +58,7 @@ def convert_upper_camel_method_to_lower_camel(method_name: str) -> str:
     prefix = method_name.split('_')[0]
     method = upper_camel_to_lower_camel(method_name.split('_')[1])
     return prefix + '_' + method
+
+
+def get_now_time_stamp():
```

Comment:
`time_stamp` or `timestamp`. which one is better?

Reply:
I'll update it to 'timestamp'.